### PR TITLE
Applied various fixes for 6.0

### PR DESCRIPTION
**Pull request by julian-cable**
Addresses these issues:
#24 
#79 
#364 
#369 
#373 
#404 
#406

### CODE REVIEW

**Comment by julian-cable** [comment]
@daobrien @nmuller66 @sffrench These style guide fixes are ready for review.

**Comment by julian-cable** [comment]
These fixes are for the upcoming October release 6.0 of the style guide.

File: en-US/Design.xml [hunk 1]
```diff
@@ -863,10 +863,12 @@ $ vi myFile.txt
...
-					<programlisting>
-					Open
-					-Shift
-					</programlisting>
+				<literallayout>
+					For advanced management capabilities with Red   
+					Hat Satellite and cloud management services, use the Red     
+					Hat Enterprise Linux Smart Management Add    
+					-On.
+				</literallayout>
```
Comment (daobrien) [inline en-US/Design.xml:871]: I'd like to see this called out better, perhaps using `<example>` elements or similar? As-is, it looks like part of the main body of the guide.

File: en-US/Grammar.xml [hunk 2]
```diff
@@ -471,10 +471,15 @@
 				</thead>
 				 <tbody>
 					<row>
-						<entry> The individual member of the social community often receives information via visual, symbolic channels. </entry>
-						 <entry> People read. (Translation by Richard Feynman.) </entry>
+						<entry> The <command>fsck</command> utility performs a file system verification on the XFS file system residing on the <filename>/dev/vdb1</filename> partition. </entry>
+						 <entry> The <command>fsck</command> utility verifies the XFS file system on the <filename>/dev/vdb1</filename> partition. </entry>
```
Comment (daobrien) [inline en-US/Grammar.xml:475]: The example is much better (I've been meaning to fix this for ages) but it's also not quite accurate. If you run `fsck` with no args it checks all file systems on all devices. If you want to check a specific file system then you have to pass it as an arg. E.g., `fsck /dev/hd1`

File: en-US/Grammar.xml [hunk 3]
```diff
@@ -471,10 +471,15 @@
...
 
 					</row>
+					<row>
+						<entry> It is possible that the scheduled snapshot takes some time to be created. </entry>
+						 <entry> Creating the scheduled snapshot might take time. </entry>
```
Comment (daobrien) [inline en-US/Grammar.xml:480]: Everything takes time. Can we elaborate a bit? Something like "Creating the snapshot will take more or less time depending on the size of the system." Or maybe something different altogether?

File: en-US/Language.xml [hunk 4]
```diff
@@ -142,6 +161,16 @@
 
 				</listitem>
 
+			</varlistentry>
+			 <varlistentry>
+				<term>boil the ocean</term>
+				 <listitem>
+					<para>
+						Do not use. State exactly what you mean, such as "increase the scope hugely".
+					</para>
```
Comment (daobrien) [inline en-US/Language.xml:170]: tbh I would never write that and I don't think we should offer it as an example. I'm also not convinced that it's a substitute for "boil the ocean". Maybe @sffrench has some ideas?

File: en-US/Language.xml [hunk 5]
```diff
@@ -265,6 +304,16 @@
...
+				 <listitem>
+					<para>
+						Do not use. Explain exactly what you mean, such as "a waste of effort".
+					</para>
+
+				</listitem>
+
 			</varlistentry>
 			 <varlistentry>
 				<term>flying by the seat of your pants</term>
```
Comment (daobrien) [inline en-US/Language.xml:319]:
```suggestion
				<term>fly by the seat of your pants</term>
```

File: en-US/Translation.xml [hunk 6]
```diff
@@ -78,8 +78,9 @@
         It is important to provide sufficient spacing between elements in your documents to facilitate reading and comprehension.
         You can include a lot of information in a few short paragraphs but readers need to be able to process that information in chunks.
         The same consideration applies to lists.
-        If you use DocBook to build itemized, ordered, or variable (definition) lists, you can use the <option>compact</option> or <option>normal</option> attributes to specify the spacing between list items.
-        DocBook uses the <option>normal</option> spacing attribute by default.
+        If you use DocBook or AsciiDoc to build itemized or ordered lists, you can choose between normal spacing or the <option>compact</option> attribute to specify the spacing between list items.
+		In AsciiDoc, compact spacing is not supported for variable (definition) lists.
+        DocBook and AsciiDoc use the <option>normal</option> spacing attribute by default.
```
Comment (daobrien) [inline en-US/Translation.xml:83]: We're in the process of removing language-specific guidelines from the style guide. We shouldn't be adding more. People who use the style guide might write in anything from LaTex to googledoc to Word or even Lotus Notes <gulp>.

**Review by daobrien — CHANGES_REQUESTED** [review]

File: en-US/Design.xml (hunk 1 above)
Comment (julian-cable) [reply en-US/Design.xml:871]: Reformatted as an example.

File: en-US/Grammar.xml (hunk 2 above)
Comment (julian-cable) [reply en-US/Grammar.xml:475]: Updated the command to include arguments.

File: en-US/Grammar.xml (hunk 3 above)
Comment (julian-cable) [reply en-US/Grammar.xml:480]: Replaced with a different example.

File: en-US/Language.xml (hunk 4 above)
Comment (julian-cable) [reply en-US/Language.xml:170]: Removed this entry; I don't think it is needed.

File: en-US/Language.xml (hunk 5 above)
Comment (julian-cable) [reply en-US/Language.xml:319]: Fixed

File: en-US/Translation.xml (hunk 6 above)
Comment (julian-cable) [reply en-US/Translation.xml:83]: Fixed

File: en-US/Translation.xml [hunk 7]
```diff
@@ -78,13 +78,13 @@
         It is important to provide sufficient spacing between elements in your documents to facilitate reading and comprehension.
         You can include a lot of information in a few short paragraphs but readers need to be able to process that information in chunks.
         The same consideration applies to lists.
-        If you use DocBook or AsciiDoc to build itemized or ordered lists, you can choose between normal spacing or the <option>compact</option> attribute to specify the spacing between list items.
+        <!-- If you use DocBook or AsciiDoc to build itemized or ordered lists, you can choose between normal spacing or the <option>compact</option> attribute to specify the spacing between list items.
 		In AsciiDoc, compact spacing is not supported for variable (definition) lists.
-        DocBook and AsciiDoc use the <option>normal</option> spacing attribute by default.
-		<!-- In AsciiDoc? -->
+        DocBook and AsciiDoc use the <option>normal</option> spacing attribute by default. -->
```
Comment (daobrien) [inline en-US/Translation.xml:83]: Unless you have specific reason for keeping it, just delete this and similar content.

**Review by daobrien — COMMENTED** [review]
I'm ok with 99% of this but recommend @psweany or maybe @bonnevil review the example for `fsck`. I'd like to make sure it's a sound example.

File: en-US/Grammar.xml (hunk 2 above)
Comment (daobrien) [reply en-US/Grammar.xml]: @psweany or @bonnevil comment please?

File: en-US/Translation.xml (hunk 7 above)
Comment (julian-cable) [reply en-US/Translation.xml]: Deleted

**Review by daobrien — APPROVED** [review]

**Pull request merged**